### PR TITLE
Rancher 468

### PR DESCRIPTION
**Pull request by shgulomov**
Resized the size of ElasticSearch disk. Configured logs retention policy in Terraform using ELK stack provider.

### CODE REVIEW

File: terraform/rancher/cluster/logging.tf [hunk 1]
```diff
@@ -108,13 +126,12 @@ data:
...
     # Send the logs to the standard output
     <match **>
-      @type elasticsearch_dynamic
+      @type elasticsearch
       include_tag_key true
       host "#{ENV['ELASTICSEARCH_HOST']}"
       port "#{ENV['ELASTICSEARCH_PORT']}"
       scheme http
       logstash_format true
-      logstash_prefix logstash-${record["kubernetes"]["namespace_name"]}
```
Comment (OHaimanov) [inline terraform/rancher/cluster/logging.tf:117]: This is Kseniia changes about filtering logs by namespace, so please bring this changes back

File: terraform/rancher/cluster/provider.tf [hunk 2]
```diff
@@ -35,3 +35,9 @@ provider "kubernetes" {
   cluster_ca_certificate = base64decode(data.aws_eks_cluster.cluster.certificate_authority.0.data)
   token                  = data.aws_eks_cluster_auth.cluster.token
 }
+
+provider "elasticstack" {
+  elasticsearch {
+    endpoints = [var.elasticsearch_url]
```
Comment (OHaimanov) [inline terraform/rancher/cluster/provider.tf:41]: This url should be generated like "${module.eks_cluster.cluster_id}-elasticsearch.${var.root_domain}"

**Review by OHaimanov — CHANGES_REQUESTED** [review]

File: terraform/rancher/cluster/provider.tf (hunk 2 above)
Comment (shgulomov) [reply terraform/rancher/cluster/provider.tf]: I've changed the URL.

File: terraform/rancher/cluster/logging.tf (hunk 1 above)
Comment (shgulomov) [reply terraform/rancher/cluster/logging.tf]: Done. Reverted my commit.

**Review by OHaimanov — APPROVED** [review]

**Review by vkapylou — APPROVED** [review]

**Comment by shgulomov** [comment]
Closing

**Pull request merged**